### PR TITLE
Add more checks to the chat event 

### PR DESCRIPTION
**Pull request by BeatrixCohere**

### CODE REVIEW

File: .readthedocs.yml
Comment (billytrend-cohere) [inline .readthedocs.yml:1]: is this required? looks unrelated

Comment (BeatrixCohere) [reply .readthedocs.yml:1]: oops

**Review by billytrend-cohere — APPROVED** [review]

**Review by daniel-cohere — APPROVED** [review]
